splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/UpdateOperationIT.java

```diff
@@ -506,6 +519,114 @@ public void testUpdateUsingBoolDescIndex() throws Exception {
         Assert.assertEquals("Incorrect number of rows updated", 1, n);
     }
 
+    @Test
+    public void testUpdateUsindIndexLookupViaControl() throws Exception {
+        TestConnection conn = methodWatcher.getOrCreateConnection();
+        conn.setAutoCommit(false);
+
+        try (Statement s = conn.createStatement()) {
+            /* check original table content */
+            String expected = "A1 |B1 |C1 |D1  |E1  |\n" +
+                    "----------------------\n" +
+                    "10 |10 |10 |III |III |\n" +
+                    " 3 | 3 | 3 |AAA |AAA |\n" +
+                    " 8 | 8 | 8 |GGG |GGG |";
+            ResultSet rs = methodWatcher.executeQuery("select * from t1");
+            assertEquals(expected, TestUtils.FormattedResult.ResultFactory.toString(rs));
+            rs.close();
+
+            /* update Q1 -- simple update*/
+            String sql = "update t1 --SPLICE-PROPERTIES index=t1_ix_c1, useSpark=false\n" +
+                    "set b1=100 where c1>5";
+            int n = s.executeUpdate(sql);
+            Assert.assertEquals("Incorrect number of rows updated", 2, n);
+
+            expected = "A1 |B1  |C1 |D1  |E1  |\n" +
+                    "-----------------------\n" +
+                    "10 |100 |10 |III |III |\n" +
+                    " 3 | 3  | 3 |AAA |AAA |\n" +
+                    " 8 |100 | 8 |GGG |GGG |";
+            rs = methodWatcher.executeQuery("select * from t1");
+            assertEquals(expected, TestUtils.FormattedResult.ResultFactory.toString(rs));
+            rs.close();
+
+            /* update Q2 -- update with SSQ */
+            sql = "update t1 --splice-properties index=t1_ix_c1, useSpark=false\n" +
+                    "set b1 = (select b2*10 from t2 where a1=a2)" +
+                    "where c1>5";
+            n = s.executeUpdate(sql);
+            Assert.assertEquals("Incorrect number of rows updated", 2, n);
+
+            expected = "A1 | B1   |C1 |D1  |E1  |\n" +
+                    "-------------------------\n" +
+                    "10 |10000 |10 |III |III |\n" +
+                    " 3 |  3   | 3 |AAA |AAA |\n" +
+                    " 8 |10000 | 8 |GGG |GGG |";
+            rs = methodWatcher.executeQuery("select * from t1");
+            assertEquals(expected, TestUtils.FormattedResult.ResultFactory.toString(rs));
+            rs.close();
+        }
+
+        // roll back the update
+        try {
+            conn.rollback();
+        } catch (Exception e) {} // Swallow for HFile Bit Running in Control
```

Comment:
Why do you have to swallow the exception?

Reply:
Thanks @dgomezferro for catching this. It is a careless copy-paste error. My purpose was just to rollback the update. I've updated the IT to correct the misleading code and comment.